resources/js/meeting_calendar.js

```diff
@@ -62,11 +63,11 @@ function createProcess(year, month) {
             } else {
                 count++;
                 var dateValue = `${year}-${month + 1}-${count}`;
-                var url = `users/meetings/search/${dateValue}/result`;
+                var url = `http://127.0.0.1:8000/users/meetings/search/${dateValue}/result`;
```

Comment:
Please don't use this http://127.0.0.1:8000. It will cause an error when deploying to production server